atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TimeLockHelperServices.java

```diff
@@ -42,10 +43,11 @@ static TimeLockHelperServices create(
             Set<Schema> schemas,
             LockWatchStarter lockWatchStarter,
             LockWatchCachingConfig lockWatchCachingConfig,
-            Supplier<Optional<RequestBatchersFactory.MultiClientRequestBatchers>> requestBatcherProvider) {
+            Supplier<Optional<RequestBatchersFactory.MultiClientRequestBatchers>> requestBatcherProvider,
+            LockWatchTimeLockDiagnosticsLogger lockWatchTimeLockDiagnosticsLogger) {
 
-        LockWatchManagerInternal lockWatchManager =
-                LockWatchManagerImpl.create(metricsManager, schemas, lockWatchStarter, lockWatchCachingConfig);
+        LockWatchManagerInternal lockWatchManager = LockWatchManagerImpl.create(
+                metricsManager, schemas, lockWatchStarter, lockWatchCachingConfig, lockWatchTimeLockDiagnosticsLogger);
```

Comment:
The name reads awkward (`lockWatchTimeLockDiagnosticsLogger`). I wanted to name it something where it's clear we are hitting the server side from the client side in an RPC.

This might not be the best name, and I personally found it tedious to read through the code between client/server side where classes were named very similarly without much to indicate which one is where.